### DISCUSSION TITLE
Global Theming

### DISCUSSION THREAD

**Discussion by ghostdevv** [Budibase/budibase#1720] [Help] [answered]
I was wondering how to globally set the theme of my app, it's possible to do it in *some* components like tables, I essentially wanna set a dark theme on my app.

**Comment by joebudi** [comment, marked as answer]
Hey. Theming is on the roadmap.  We're currently porting components to our new design system. Once that is done, apps will have 4 base themes (lightest, light, dark, darkest) like the builder 🙂. I'm not sure exactly when this will be delivered but I hope it will come in the July release.

**Reply by ghostdevv** [reply]
Amazing thank you
